.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+jsconfig.json
+.eslintrc
+.vscode
+/coverage/
```

Comment:
These comments will reveal my obsessive nature:

  - The final line should be terminated by a newline character (`\n`). You may like to [configure your editor][1] to insert this automatically.

  - I like to begin all paths with `/` so it's clear exactly where we expect a file or directory to reside. (There are situations in which this is not practical.)

  - I like to end paths to directories with `/`.

  - I like to list directories before files, and order the paths in each of these groups alphabetically.

  - Only files created by the project should be listed. On macOS, for example, directories may contain __.DS_Store__ files. One never wants to commit these, so they should be listed in one's [global .gitignore][2]. __.vscode__ falls into this category.

  - I believe __jsconfig.json__ and __.eslintrc__ could be removed from the list.


[1]: https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support
[2]: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore